src/video/canvas.cpp

```diff
@@ -127,7 +127,9 @@ Canvas::draw_surface(const SurfacePtr& surface,
   request->blend = blend;
 
   request->srcrects.emplace_back(Rectf(surface->get_region()));
-  request->dstrects.emplace_back(Rectf(apply_translate(position), Size(surface->get_width(), surface->get_height())));
+  request->dstrects.emplace_back(Rectf(apply_translate(position)*scale(),
```

Comment:
Add spaces around the `*`

Reply:
I've rarely seen so many consecutive closing parentheses.